dotnet-logging/LogLevelsWithSerilog/LogLevelsWithSerilog/LogLevelsWithSerilog.csproj

```diff
@@ -1,7 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
 
 	<PropertyGroup>
-		<TargetFramework>net6.0</TargetFramework>
+		<TargetFramework>net7.0</TargetFramework>
+		<LangVersion>10</LangVersion>
```

Comment:
We can remove this, .NET 7.0 defaults to C# 11

Reply:
updated